runtime-interface/src/lib.rs

```diff
@@ -47,6 +56,9 @@ pub use gear_sandbox::sandbox;
 
 const _: () = assert!(core::mem::size_of::<HostPointer>() >= core::mem::size_of::<usize>());
 
+// Domain Separation Tag for signatures on G2.
+pub const DST_G2: &[u8] = b"BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_";
```

Comment:
Are you sure this value will not be changed in future? I mean may be better to provide it from runtime

Reply:
Nope, this constant is from the milagro_bls library. And if we ever need to change it we will introduce just a new function.